src/Hudl.FFmpeg.Core/Attributes/AttributeRetrieval.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
```

Comment:
It's pretty tough to tell what all changed in the conversion to .NET Standard - could all of these be file moves instead of a second copy of the source?